src/js/tabs/pid_tuning.js

```diff
@@ -2697,6 +2697,10 @@ TABS.pid_tuning.changeRatesSystem = function(sameType) {
                 rc_rate_pitch_e.val((FC.RC_TUNING.rcPitchRate * 1000).toFixed(0));
                 rc_rate_e.val((FC.RC_TUNING.RC_RATE * 1000).toFixed(0));
                 rc_rate_yaw_e.val((FC.RC_TUNING.rcYawRate * 1000).toFixed(0));
+            } else if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+                rcRateDefault = (70).toFixed(0);
+                rateDefault = (670).toFixed(0);
+                expoDefault = (0).toFixed(2);
```

Comment:
We have default values in this file? I didn't notice it before. I thought all the defaults were at fc.js.

Reply:
yes it is designed like that in the @fgiudice98 original code for Rate types.

Reply:
Thougth that we never had Rate defaults in fc.js

Reply:
I don't like the way it is, but changing it now is outside of the scope of this PR, so to me is ok.

Reply:
Maybe we can made this change for next PR's yes, i think it make sense.